docs/scarpet/api/Auxiliary.md

```diff
@@ -42,89 +56,105 @@ Throws `unknown_particle` if particle doesn't exist.
+Shapes will fail to draw and raise a runtime error if not all their required parameters
 are specified and all available shapes have some parameters that are required, so make sure to have them in place:
 
-On the client, shapes can recognize that they are being redrawn again with the same parameters, disregarding the 
-duration parameter. This updates the expiry on the drawn shape to the new value, instead of adding new shape in its 
-place. This can be used for toggling the shapes on and off that has been send previously with very large durations, 
+On the client, shapes can recognize that they are being redrawn again with the same parameters, disregarding the
+duration parameter. This updates the expiry on the drawn shape to the new value, instead of adding new shape in its
+place. This can be used for toggling the shapes on and off that has been send previously with very large durations,
 or simply refresh the shapes periodically in more dynamic applications.
 
 Optional shared shape attributes:
- * `color` - integer value indicating the main color of the shape in the form of red, green, blue and alpha components 
- in the form of `0xRRGGBBAA`, with the default of `-1`, so white opaque, or `0xFFFFFFFF`.
- * `player` - name or player entity to send the shape to, or a list of players. If specified, the shapes will appear only for the specified
- players (regardless where they are), otherwise it will be send to all players in the current dimension.
- * `line` - (Deprecated) line thickness, defaults to 2.0pt. Not supported in 1.17's 3.2 core GL renderer.
- * `fill` - color for the faces, defaults to no fill. Use `color` attribute format
- * `follow` - entity, or player name. Shape will follow an entity instead of being static.
-   Follow attribute requires all positional arguments to be relative to the entity and disallow
-   of using entity or block as position markers. You must specify positions as a triple.
- * `snap` - if `follow` is present, indicated on which axis the snapping to entity coordinates occurs, and which axis
-   will be treated statically, i.e. the coordinate passed in a coord triple is the actual value in the world. Default
-   value is `'xyz'`, meaning the shape will be drawn relatively to the entity in all three directions. Using `xz` for 
-   instance makes so that the shape follows the entity, but stays at the same, absolute Y coordinate. Preceeding an axis
-   with `d`, like `dxdydz` would make so that entity position is treated discretely (rounded down).
- * `debug` - if True, it will only be visible when F3+B entity bounding boxes is enabled.
- * `facing` - applicable only to `'text'`, `'block'` or '`item'` shapes, where its facing. Possible options are:
-   * `player`: Default. Element always rotates to face the player eye position, 
-   * `camera`: Element is placed on the plane orthogonal to player look vector, 
-   * `north`, `south`, `east`, `west`, `up`, `down`: obvious
```

Comment:
This shouldn't have been removed, I think it's because they were added after the PR was started.